core/src/main/java/org/elasticsearch/action/bulk/BulkShardResponse.java

```diff
@@ -51,6 +52,16 @@ public ShardId getShardId() {
     }
 
     @Override
+    public void setForcedRefresh(boolean forcedRefresh) {
```

Comment:
I haven't looked into the impl yet but does that mean every bulk causes a refresh? If so I think we should rethink this? if not I think we should document it here.